catalyst/contrib/datasets/mnist.py

```diff
@@ -277,5 +345,15 @@ def query_size(self) -> int:
         """Query Gallery dataset should have query_size property"""
         return self._query_size
 
+    @property
+    def data(self) -> torch.Tensor:
```

Comment:
why do we need it?

Reply:
why not?
i think it's ok when you can get data and target from your dataset directly without using private fields
moreover, it gives us some coherence in mnist datasets (train mnist dataset has these properties, classic mnist has too)